src/Microsoft.VisualStudio.SlowCheetah/Transformer/JsonTransformer.cs

```diff
@@ -79,22 +119,21 @@ public bool Transform(string source, string transform, string destination)
             }
         }
 
-        private bool TrySaveToFile(Stream result, string destinationFile)
+        private bool TrySaveToFile(Stream result, string sourceFile, string destinationFile)
```

Comment:
We probably want to revert this change as you found `WriteAllText` destroys the files encoding. Then switch to using your new `GetEncoding` helper.